index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+   <section class="full-bgcolor">
+    <header class="header">
+        <nav class="navbar">
+           <a class="header-logo" href="#">My Logo</a>
+           <div class="menu-icon">
+           <a class="menubar" href="#"><img src="/images/Enabled.png" alt="menu" /></a>
+           </div>
+        </nav>
+    </header>
+    <main class="main-content">
+        <section class="headline">
+            <h1 class="headline-title">I am Md Ariful Glad to see you!</h1>
+            <p class="headline-content">I'm a software developer I can help you
+                build a product feature or website Look through some of my works and
+                experience! If you like what you see and have a project you need coded
+                don't hesitate to contact me.
+            </p>
+         <div class="headline-social">
+            <h3 class="social-title">LET'S CONNECT</h3>
```

Comment:
- [ ] Great job! :rocket: On matching the Figma design, but there's just one slight problem. Currently, you're not using the correct font for your content. I recommend using _Poppins_. That way you'll have a much prettier looking UI.

Reply:
Hello reviewer I update my Readme file as requirement in MIT licence link, Auther section proper link, used Poppins font in css file please recheck my pull request.